images/macos/templates/macOS-11.pkr.hcl

```diff
@@ -0,0 +1,242 @@
+    source      = "./tests"
+  }
+  provisioner "file" {
+    destination = "image-generation/"
+    source      = "./software-report"
+  }
+  provisioner "file" {
+    destination = "image-generation/"
+    source      = "./helpers"
+  }
+  provisioner "file" {
+    destination = "image-generation/add-certificate.swift"
+    source      = "./provision/configuration/add-certificate.swift"
+  }
+  provisioner "file" {
+    destination = ".bashrc"
+    source      = "./provision/configuration/environment/bashrc"
+  }
+  provisioner "file" {
+    destination = ".bash_profile"
+    source      = "./provision/configuration/environment/bashprofile"
+  }
+  provisioner "file" {
+    destination = "./"
+    source      = "./provision/utils"
+  }
+  provisioner "shell" {
+    inline = [
+      "mkdir ~/bootstrap"
+    ]
+  }
+  provisioner "file" {
+    destination = "bootstrap"
+    source      = "./provision/bootstrap-provisioner/"
+  }
+  provisioner "file" {
+    destination = "image-generation/toolset.json"
+    source      = "./toolsets/toolset-11.json"
+  }
+  provisioner "shell" {
+    scripts = [
+      "./provision/configuration/add-network-interface-detection.sh",
+      "./provision/configuration/autologin.sh",
+      "./provision/configuration/disable-auto-updates.sh",
+      "./provision/configuration/screensaver-off.sh",
+      "./provision/configuration/ntpconf.sh",
+      "./provision/configuration/max-files.sh",
+      "./provision/configuration/shell-change.sh"
+    ]
+    environment_vars = [
+      "PASSWORD=${var.macos_password}",
+      "USERNAME=${var.macos_user}"
+    ]
+    execute_command  = "chmod +x {{ .Path }}; source $HOME/.bash_profile; sudo {{ .Vars }} {{ .Path }}"
+  }
+  provisioner "shell" {
+    scripts = [
+      "./provision/configuration/preimagedata.sh",
+      "./provision/configuration/configure-ssh.sh",
+      "./provision/core/xcode-clt.sh",
+      "./provision/configuration/configure-machine.sh"
+    ]
+    environment_vars = [
+      "IMAGE_VERSION=${var.build_id}",
+      "IMAGE_OS=${var.image_os}"
+    ]
+    execute_command  = "chmod +x {{ .Path }}; source $HOME/.bash_profile; {{ .Vars }} {{ .Path }}"
+  }
+  provisioner "shell" {
+    script = "./provision/core/reboot.sh"
+    execute_command  = "chmod +x {{ .Path }}; source $HOME/.bash_profile; sudo {{ .Vars }} {{ .Path }}"
+    expect_disconnect = true
+  }
+  provisioner "shell" {
+    scripts = [
+      "./provision/core/homebrew.sh",
+      "./provision/core/powershell.sh",
+      "./provision/core/dotnet.sh",
+      "./provision/core/python.sh",
+      "./provision/core/azcopy.sh",
+      "./provision/core/openssl.sh",
+      "./provision/core/ruby.sh",
+      "./provision/core/rubygem.sh",
+      "./provision/core/git.sh",
+      "./provision/core/node.sh"
+    ]
+    execute_command  = "chmod +x {{ .Path }}; source $HOME/.bash_profile; {{ .Vars }} {{ .Path }}"
+    environment_vars = ["AGENT_TOOLSDIRECTORY=${var.tools_dir}"]
```

Comment:
Why do we need to pass this variable here and define it on the top of file?
We don't do that in current templates.

P.S. I think it was done some time ago when we had an issue with bash_profile loading. Currently, it can be removed